cfme/tests/configure/test_access_control.py

```diff
@@ -525,6 +533,7 @@ def test_group_crud_with_tag(provider, tag_value, group_collection):
 
     path = 'VM_Template-Folder' if provider.key == 'vsphere55' else 'Discovered virtual machine'
```

Comment:
Suggestion: @apagac I know this is out of scope for this PR, but I think we can remove the `vsphere55` check as the provider is deprecated. What do you think?

Reply:
At least add a TODO here for now please